prism-agent/service/server/src/main/scala/io/iohk/atala/verification/controller/http/VcVerificationRequest.scala

```diff
@@ -53,12 +54,16 @@ object VcVerificationRequest {
 
   given credentialVerificationRequestSchema: Schema[VcVerificationRequest] = Schema.derived
 
-  def toService(request: VcVerificationRequest): List[ServiceVcVerificationRequest] = {
-    request.verifications.map(verification =>
-      ServiceVcVerificationRequest(
-        credential = request.credential,
-        verification = VcVerification.convert(verification.verification),
-        parameter = verification.parameter.map(param => VcVerificationParameter.convert(param))
+  def toService(request: VcVerificationRequest): IO[ErrorResponse, List[ServiceVcVerificationRequest]] = {
+    ZIO.collectAll(
+      request.verifications.map(verification =>
+        for {
+          serviceVerification <- VcVerification.convert(
+            verification.verification,
```

Comment:
it would be nice to have a different names for the object (verification) and internal field (verification).

Reply:
will fix it in follow up